apps/web/frontend/components/projectMemberList/projectMemberList.tsx

```diff
@@ -53,24 +53,11 @@ export const ProjectMemberList = (props: ProjectMemberListProps) => {
       <tbody>
         {project.members.map((user) => (
           <tr key={user.id}>
-            <td className="flex w-full items-center gap-2">
-              {user.image ? (
-                <div className="avatar">
-                  <Image
-                    className="rounded-box"
-                    width={64}
-                    height={64}
-                    src={user.image}
-                    alt={user.name ?? 'image of the user'}
-                  />
-                </div>
-              ) : (
-                <div className="avatar placeholder">
-                  <div className="size-16 rounded-box bg-neutral text-neutral-content" />
-                </div>
-              )}
-              {user.name}
-              <ProjectRoleLabel role={user.projectRole} />
+            <td className="flex items-center gap-2">
+              <div className="min-w-52">
+                <UserLabel name={user.name ?? user.id} image={user.image ?? undefined} imageSize={28} />{' '}
+              </div>
```

Comment:
```suggestion
                <UserLabel name={user.name ?? user.id} image={user.image ?? undefined} imageSize={28} />{' '}
```

Same as org member list

Reply:
We added the min-w-32 so the role label has a constant space to the name.
![image](https://github.com/user-attachments/assets/3b3c5b5f-abbc-4500-b4b9-13b912028fdb)